src/parser/core/modules/DisengageGcds.tsx

```diff
@@ -0,0 +1,53 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Action} from 'data/ACTIONS'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import {SimpleStatistic, Statistics} from 'parser/core/modules/Statistics'
+import React from 'react'
+
+export class DisengageGcds extends Analyser {
+	static override handle = 'disengagegcds'
+
+	@dependency data!: Data
+	@dependency statistics!: Statistics
+
+	private disengageGcds: number = 0
+
+	// Pass in the GCD you want to track as an Action.
+	protected trackedAction: Action = this.data.actions.UNKNOWN
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		this.addEventHook(playerFilter.type('action').action(this.trackedAction.id), this.onUse)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	// Override this to suggest job-specific ways to handle downtime
+	protected addDisengageSuggestion() {
+		this.statistics.add(new SimpleStatistic({
+			title: <Trans id="core.disengage.statistic.title">
+				{this.trackedAction.name} Uses
+			</Trans>,
+			icon: this.trackedAction.icon,
+			value: `${this.disengageGcds}`,
+			info: <Trans id="core.disengage.statistic.info">
```

Comment:
Action; It would be better if the info i18n string was declared outside of the protected function so that you can just override the string, instead of having to override the entire function just to change some text. This would keep it inline with other core modules.

Reply:
Thanks again for reviewing, I have updated to make the info string configurable.  Is it useful to do the same with the title, and if so how?

My instinct here would be that we'd want to default to using the name of thedisengage ability we're telling people to avoid in the title, since that's shorter and more specific than something like "avoid your basic ranged attack". 

From what I can tell though, we need to provide the title as an override to the Analyser module, and that this must be a text string rather than a ReactNode. Is there a way to pass in a variable to the t() function? I couldn't find an example with a little bit of rummaging around, and this doesn't seem to work:

```
	static override title = t('core.disengage.statistic.title')`{this.trackedAction.name} Uses`
```

Would we want to:

1. provide an "[action name] uses]" title within the function?
2. provide a title string like "Ranged Attack Uses" which can be overridden, but which is less specific by default?
3. a secret third thing that hasn't occurred to me or that I don't know how to do as a Typescript newbie?

Reply:
I would probably vote for Ranged Attack Uses and allow override, seems simple enough for implementers.